backend/src/main/java/com/bakdata/conquery/models/preproc/Preprocessed.java

```diff
@@ -80,7 +72,10 @@ public Preprocessed(ConqueryConfig config, PreprocessingJob preprocessingJob) th
 		for (int index = 0; index < input.getWidth(); index++) {
 			ColumnDescription columnDescription = input.getColumnDescription(index);
 			columns[index] = new PPColumn(columnDescription.getName(), columnDescription.getType());
-			columns[index].setParser(columnDescription.getType().createParser(config));
+
+			//ToDo 1-)createParser aus der OutPutDescription
+
+			columns[index].setParser(input.createParser(index, config));
```

Comment:
würde den zugriff auf die `output`s unrollen, die Methode hat keinen nutzen außer hier und ein Zugriff über den Index ist relativ leaky als wiederverwendbare API

Reply:
in createParser machst du `output[index].createParser` oder so ähnlich, das würde ich einfach an der Stelle machen. Es ist zwar nicht 100% Law-of-Demeter konform aber besser als eine super spezielle Methode dort zu implementieren.